media-proxy/src/mesh/st2110_20rx.cc

```diff
@@ -0,0 +1,88 @@
+#include "st2110rx.h"
+
+namespace mesh::connection {
+
+ST2110_20Rx::ST2110_20Rx() {}
+
+ST2110_20Rx::~ST2110_20Rx() {}
+
+st_frame *ST2110_20Rx::get_frame(st20p_rx_handle h) { return st20p_rx_get_frame(h); };
+
+int ST2110_20Rx::put_frame(st20p_rx_handle h, st_frame *f) { return st20p_rx_put_frame(h, f); };
+
+st20p_rx_handle ST2110_20Rx::create_session(mtl_handle h, st20p_rx_ops *o)
+{
+    return st20p_rx_create(h, o);
+};
+
+int ST2110_20Rx::close_session(st20p_rx_handle h) { return st20p_rx_free(h); };
+
+Result ST2110_20Rx::configure(context::Context &ctx, const std::string &dev_port,
+                              const MeshConfig_ST2110 &cfg_st2110,
+                              const MeshConfig_Video &cfg_video)
+{
+    static int session_id;
+    static std::mutex mtx;
+    std::lock_guard<std::mutex> lock(mtx);
+
+    if (cfg_st2110.transport != MESH_CONN_TRANSPORT_ST2110_20) {
+        set_state(ctx, State::not_configured);
+        return set_result(Result::error_bad_argument);
+    }
+
+    _st = get_mtl_handle(dev_port, MTL_LOG_LEVEL_CRIT, cfg_st2110.local_ip_addr);
+    if (_st == nullptr) {
```

Comment:
You initialize `_st` with `0`. Why do check it to be `nullptr` here then? Avoid ambiguity.

Reply:
Initialized with ```nullptr```
Fixed in https://github.com/OpenVisualCloud/Media-Communications-Mesh/pull/261/commits/2479ca6f6d060db56009026a4c727be98472c6dc